api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -302,7 +313,10 @@ public WorkspaceACL getWorkspaceAcl(String projectName, String workspaceName) {
   @Override
   public WorkspaceResponse getWorkspace(String projectName, String workspaceName) {
```

Comment:
Agreed. Worth thinking about. I'm not entirely sure that its an abstraction that provides enough value to do the refactor? Might warrant a bit of time thinking about though.

Reply:
Once we get immutables, it might be worth it, because it's so easy to make little single-purpose classes. Don't need to worry about it for now though.

Reply:
Agreed, this is one of the reasons I wanted to adopt Kotlin. It makes it incredibly easy to introduce small data classes like this.